packages.json

```diff
@@ -7200,5 +7200,28 @@
     "description": "csvql.",
     "license": "MIT",
     "web": "https://github.com/Bennyelg/csvql"
+  },
+  {
+    "name": "contracts",
+    "url": "https://github.com/Udiknedormin/NimContracts",
+    "method": "git",
+    "tags": [
+      "library",
+      "pure",
+      "contracts",
+      "design by contract",
```

Comment:
no spaces in tags

Reply:
Doesn't patty package have spaces in its tags?